scanners/a11y.py

```diff
@@ -48,90 +47,34 @@ def cache_is_not_forced(options):
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H39',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H42',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H43',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H44',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H48',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H49',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H63',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H65',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H71',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H73',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H85',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H93',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.F68',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H39.3.LayoutTable',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H43.HeadersRequired',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H71.NoLegend',
-                'WCAG2AA.Principle1.Guideline1_3.1_3_1.H73.3.LayoutTable',
-                'WCAG2AA.Principle2.Guideline2_2.2_2_1.F41.2',
-                'WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID',
-                'WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1',
-                'WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.EmptyTitle',
-                'WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.3.Lang',
-                'WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.3.XmlLang',
-                'WCAG2AA.Principle3.Guideline3_1.3_1_2.H58.1.Lang',
-                'WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2'
-            ]
-        }
-    }
-
-    payload = json.dumps(lambda_options).encode()
-
-    response = client.invoke(
-        FunctionName=os.environ['AWS_LAMBDA_PA11Y_FUNCTION_NAME'],
-        Payload=payload,
-    )
-
-    response_payload_bytes = response['Payload'].read()
-    response_payload_string = response_payload_bytes.decode('UTF-8')
-    response_payload_json = json.loads(response_payload_string)
-
-    logging.debug("Invoking a11y_lambda function: %s" % lambda_options)
-
-    results = response_payload_json
-    errors = get_errors_from_results(results)
+def cache_errors(errors, domain, cache):
     cachable = json.dumps({'results': errors})
     logging.debug("Writing to cache: %s" % domain)
     content = cachable
     destination = cache
     utils.write(content, destination)
-    return errors
 
 
-def get_errors_from_results(results):
-    errors = []
-    for result in results:
-        if result['type'] == 'error':
-            errors.append(result)
-    return errors
+def run_a11y_scan(domain, cache):
+    logging.debug("[%s][a11y]" % domain)
+    pa11y = os.environ.get("PA11Y_PATH", "pa11y")
+    command = [pa11y, domain, "--reporter", "json", "--config", "pa11y_config.json", "--level", "none", "--timeout", "300000"]
```

Comment:
This makes me wonder if `pa11y_config.json` should live in the Pulse repo, since that's the project which is setting the configuration and parameters for this scan. You could then pass in a raw.github.com URL to that JSON file to the scan environment, much like various scanners in this project accept either a local or remote URL to content stored elsewhere.